src/components/CrudContainer/CrudContainer.test.tsx

```diff
@@ -2,29 +2,35 @@ import { render, waitFor } from "@testing-library/react"
 import { CrudContainer } from "./CrudContainer"
 import userEvent from '@testing-library/user-event'
 
+const default_props = {
+    wines: [],
+    setWines: () => []
+}
```

Comment:
nice!

Could you change this to camelCase?

I wonder if it would be helpful to make the fields readonly using a Typescript interface. This would prevent `default_props` from being editing between tests